tests/stub/routing.py

```diff
@@ -1398,7 +1521,7 @@ def work(tx):
 
     def test_should_use_initial_router_for_discovery_when_others_unavailable(self):
         # TODO add support and remove this block
-        if get_driver_name() in ['python', 'javascript', 'go', 'dotnet']:
+        if get_driver_name() in ['python', 'javascript', 'go', 'dotnet', 'java']:
```

Comment:
Please remember to remove this in the next PR.

Reply:
I also added this one explicitly